pages/index.vue

```diff
@@ -14,6 +14,13 @@
       "
       >Create user</VBtn
     >
+    <CompanyCard
```

Comment:
CompanyCard should be passed as child or props. See the above comment attached to this file